src/Config.php

```diff
@@ -55,7 +55,8 @@ public function get($configKey)
             throw new \InvalidArgumentException('Configuration key must not be empty', 1444039407);
         }
         // Extract parts of the path
-        $configKey = str_getcsv($configKey, '.');
+        $configKey = str_getcsv($configKey, '.',  '"', '\\');
```

Comment:
how is this change related?

Reply:
See commit msg:

> Additional the call to str_getcsv is adapted to set all optional
arguments as the defaults will change in PHP >= 9 and PHP 8.4
deprecated the omission of the optional arguments.

Reply:
Thanks!